webpack/webpack.config.dll.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   entry: {
     vendors: [
       'aphrodite-local-styles',
+      'auth0-js',
       'auth0-lock',
```

Comment:
remove?

Reply:
No - we're not there yet! Still need signup and password recovery.